LayoutTests/http/tests/paymentrequest/ApplePayModifier-disbursementPaymentRequest.https.html

```diff
@@ -0,0 +1,116 @@
+<!doctype html>
+<meta charset="utf-8" />
+<title>
+  Tests for providing `disbursementPaymentRequest` as part of
+  `ApplePayModifier`.
+</title>
+<script src="/js-test-resources/ui-helper.js"></script>
+<script src="/resources/payment-request.js"></script>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+  <script>
+    setup({ explicit_done: true, explicit_timeout: true });
+
+    test((test) => {
+      const method = validPaymentMethod();
+      method.data.features = ["paymentRequestDisbursements"];
+      new PaymentRequest([method], validPaymentDetails());
```

Comment:
Is the expectation here that doesn't throw? (i.e., it's not clear what this is testing)